owncloudApp/src/main/res/values-it/strings.xml

```diff
@@ -397,9 +396,7 @@
   <string name="log_progress_dialog_text">Caricamento dati in corso...</string>
   <string name="actionbar_select_all">Seleziona tutto</string>
   <string name="actionbar_select_inverse">Seleziona inverso</string>
-  <string name="saml_authentication_required_text">Autenticazione richiesta</string>
-  <string name="saml_authentication_wrong_pass">Password errata</string>
-  <string name="actionbar_move">Sposta</string>
+    <string name="actionbar_move">Sposta</string>
```

Comment:
Same here

Reply:
Reason in https://github.com/owncloud/android/pull/2647#discussion_r319023746